doc/reference/tt_cli/configuration.rst

```diff
@@ -0,0 +1,98 @@
+tt CLI configuration
+====================
+
+.. _tt-config_file:
+
+Configuration file
+------------------
+
+The ``tt`` configuration file defines aspects of its runtime, such as directories
```

Comment:
The configuration file defines the configuration of the current environment. It defines (will define) not only the runtime instance. It's easier to talk with your voice.